front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -44,6 +44,18 @@ class PipelineController {
     pipelineDAO.getPipelinesByApplication(application)
   }
 
+  @RequestMapping(value = 'getById/{id:.+}', method = RequestMethod.GET)
```

Comment:
I agree on the query path. However I think I need to add a new prefix because otherwise I think the URLs are ambiguous.
GET /pipelines/{application}
GET /pipelines/{id}/history

Isnt GET/pipelines/{application}/{id} ambiguous with history where id=history? I know that history inst a valid id, but does the matching mechanism know that?

I went with getById thinking I was consistent with the DELETE /pipelines/deleteById.

There is a DELETE /pipelines/{application}/{pipeline}. Ideally the GET would match that.
Can we move the history URL or is that in active use?